pages/content_packs.rst

```diff
@@ -0,0 +1,101 @@
+*************
+Content Packs
+*************
+
+What are content packs?
+=======================
+
+Content packs are a convenient way to share configuration. A content pack is a
+JSON file which contains a set of configurations of Graylog components.
+This JSON file can be uploaded to Graylog instances and then installed.
+A user who took the time to create a input, pipelines and dashboard for a
+certain type of log format, can so easily share his efforts with the community.
+
+Content packs can be found on the `Graylog Marketplace <https://marketplace.graylog.org/>`__.
+
+Parameter
+---------
+
+Content packs can have parameters. Those parameters help to adjust the
+configuration to the needs of the user. A good example usage is the port of
+an input. The creator of the content pack may have his input running on port
+55055, but the user of the content pack may have already an input running on
+that specific port.
+The creator can specify a parameter and assign it to the port.
+The user of the content pack will be asked for a value of the parameter on
+installation. The value than will be used as the port of the input.
+
+How do I create a Content Pack?
+===============================
+
+1. Navigate to **System / Content Pack**.
+2. Click on **Create a Content Pack** on the upper right side of the page.
+   You will now see the wizard for a content pack creation. On the left side
+   of that page you can see the navigation of the wizard. There are 3 steps to
+   content pack creation: **Content Selection**, **Parameters** and **Preview**.
+   In the middle part of the page is the form of each wizard step. And on the
+   right side you can see the summary of your content pack.
+
+.. image:: /images/content_pack_selection.png
+
+3. Fill out the general information of the content pack.
+4. In **Content Selection** you can select the configuration you want to
+   include in your content pack. Necessary dependencies will be included
+   automatically.
+
+.. Warning:: The dependency resolution of pipeline rules does not work yet.
```

Comment:
Maybe a little too honest. I don't think marketing would like that characterization, no matter how accurate it may be. Let's put a bit of a "positive spin" on it.

Maybe, "Warning:: The one exception to this rule is dependencies for pipeline rules. Currently, grok patterns and lookup tables for pipeline rules must be added manually to the content pack. Support for automatic inclusion of pipeline rule dependencies will be added in a future release.